library/Fission/Web/Auth/Types.hs

```diff
@@ -4,13 +4,14 @@ module Fission.Web.Auth.Types
   , HerokuAddOnAPI
   ) where
 
+import Database.Esqueleto
 import Servant (BasicAuth)
 
 import Fission.Prelude
-import Fission.User.Types
+import Fission.Models
 
 -- | Authorization check for a normal user
-type ExistingUser = BasicAuth "existing user" User
+type ExistingUser = BasicAuth "existing user" (Entity User)
```

Comment:
`Entity` means "with the primary key, too". Persistent splits out the stuff that you can construct to insert a new row from the ID, and then just wraps existing rows in `Entity` to make space for the ID.